### PR TITLE
[zh-cn] Update 'styling lists'

### PR DESCRIPTION
**Pull request by jasonren0403**

### CODE REVIEW

File: files/zh-cn/learn/css/styling_text/styling_lists/index.md [hunk 1]
```diff
@@ -203,24 +215,24 @@ ul li {
...
+  - {{cssxref("background-position")}}：这定义了所选元素背景中的图像将出现在哪里——在我们的示例中设置为 `0 0`，这意味着项目符号将出现在每个列表项的最左上侧。
+  - {{cssxref("background-size")}}：设置背景图片的大小。理想条件下，我们想要项目符号与列表项的大小相同（比列表项稍大或稍小亦可）。我们使用的尺寸为 `1.6rem`（`16px`），它非常吻合我们为项目符号设置的 `20px` 的填充，16px 加上 4px 的空格间距，可以使项目符号和列表项文本效果更好。
+  - {{cssxref("background-repeat")}}：默认条件下，背景图片不断复制直到填满整个背景空间，在我们的示例中，背景图片只需复制一次，所以我们设置值为 `no-repeat`。
 
 效果显示如下：
 
-![an unordered list with the bullet points set as little star images](image-bullets.png)
+![一个无序列表，其项目符号设置为了小星星的图片](list_formatting.png)
 
 ### list-style 速记
```
Comment (yin1999) [inline files/zh-cn/learn/css/styling_text/styling_lists/index.md:233]:
```suggestion
### list-style 简写
```

File: files/zh-cn/learn/css/styling_text/styling_lists/index.md [hunk 2]
```diff
@@ -203,24 +215,24 @@ ul li {
...
 
-上述提到的三种属性可以用一个单独的速记属性 {{cssxref("list-style")}} 来设置。例如：
+上述提到的三种属性可以用一个单独的速记属性 {{cssxref("list-style")}} 来设置。例如，以下 CSS：
```
Comment (yin1999) [inline files/zh-cn/learn/css/styling_text/styling_lists/index.md:235]:
```suggestion
上述提到的三种属性可以用一个单独的简写属性 {{cssxref("list-style")}} 来设置。例如，以下 CSS：
```

File: files/zh-cn/learn/css/styling_text/styling_lists/index.md [hunk 3]
```diff
@@ -263,105 +277,142 @@ ul {
 
 ### reversed
 
-[`reversed`](/zh-CN/docs/Web/HTML/Element/ol#reversed) 属性将启动列表倒计数。示例如下：
+[`reversed`](/zh-CN/docs/Web/HTML/Element/ol#reversed) 属性将启动列表倒计数。以下示例：
```
Comment (yin1999) [inline files/zh-cn/learn/css/styling_text/styling_lists/index.md:280]:
```suggestion
[`reversed`](/zh-CN/docs/Web/HTML/Element/ol#reversed) 属性将使列表反向计数。以下示例：
```

**Review by yin1999 — APPROVED** [review]
Thanks. Just some nits

File: files/zh-cn/learn/css/styling_text/styling_lists/index.md [hunk 4]
```diff
@@ -263,105 +277,142 @@ ul {
...
 ```
 
 输出的结果如下：
 
 {{ EmbedLiveSample('value', '100%', 150) }}
 
-> **备注：** 纵然你使用非数字的 {{cssxref("list-style-type")}}, 你仍需要使用与数值同等意义的值作为 value 的属性。
+> **备注：** 即使使用非数字的 {{cssxref("list-style-type")}}，仍需要在 `value` 属性中使用等效的数值。
 
 ## 主动学习：为嵌套式列表添加样式
```
[the remark this answers is not in the document]
Comment (jasonren0403) [reply files/zh-cn/learn/css/styling_text/styling_lists/index.md:320]:
```suggestion
## 动手学习：为嵌套式列表添加样式
```

**Pull request merged**